packages/manager/test/lib.test.ts

```diff
@@ -77,6 +101,9 @@ describe('3ID Manager', () => {
 
     // twitter & github migrated
     const aka = await dataStore.get('alsoKnownAs', did)
+    const aka2 = await manager.dataStore.get('alsoKnownAs', did)
```

Comment:
@PaulLeCam any ideas what could cause this with the tileloader, everything works without cache enabled, then returns null when cache enabled, this tests for the AlsoKnownAs written during migration. While using cache all other streams appear to be written to without problem, just this single case. Still looking into it, but havent found out what it could be yet.

Reply:
I'm not sure this is the actual issue you're seeing but digging into the DID DataStore code I realized the cache doesn't get updated after record updates, so maybe this is the behavior you're seeing?
I made the changes in https://github.com/ceramicstudio/js-glaze/pull/104 and releases v0.2.2, could you check if using this version makes any difference please?